packages/esm-form-entry-app/src/app/openmrs-api/program-resource.service.ts

```diff
@@ -93,25 +83,23 @@ export class ProgramResourceService {
     );
   }
 
-  public discontinuePatientFromCareProgram(discontinuationDate: string, encounterUuid: string) {
-    const { enrollmenrDetails: enrollmentDetails } = this.singleSpaService.getPropOrThrow('additionalProps');
-
+  public discontinuePatientFromCareProgram(discontinuationDate: string, encounterUuid: string, utcOffset: string) {
+    const { enrollmenrDetails } = this.singleSpaService.getPropOrThrow('additionalProps');
     const currentDateTime = new Date();
     const hour = currentDateTime.getHours();
     const minutes = currentDateTime.getMinutes();
     const seconds = currentDateTime.getSeconds();
 
-    const discontinuationDateTime = dayjs(discontinuationDate ?? new Date())
+    const discontinuationDateTime = moment(discontinuationDate ?? new Date())
       .set('hour', hour)
       .set('minute', minutes)
-      .set('second', seconds)
-      .toISOString();
+      .set('second', seconds);
     const payload = {
-      dateEnrolled: dayjs(enrollmentDetails.dateEnrolled).toISOString(),
-      dateCompleted: discontinuationDateTime,
+      dateEnrolled: moment(enrollmenrDetails.dateEnrolled).utcOffset(utcOffset).format(),
```

Comment:
Is `dayjs(enrollmentDetails.dateEnrolled).utcOffset(utcOffset).format()` functionally equivalent here?